src/passwordHashing/generateHashPass.ts

```diff
@@ -0,0 +1,16 @@
+import * as  bcrypt from "bcrypt"
+import * as dotenv from "dotenv";
+
+//this script generate your hashed passord , change yourPassword , run the script then store -
+// the hashed value returned in the console in your .env file as PASSWORD
+dotenv.config();
+const yourPassword = "adminadmin"
+const hashPassword = async (yourPassword:string) => {
+    try {
+        const yourHashedPassword = await bcrypt.hash(yourPassword,10)
+        console.log({yourHashedPassword})
```

Comment:
Should we use `logger` here instead of `console.log` since you have defined a logging mechanism now?